### PR TITLE
ArrowSchemaConverter unit test case added

### PR DESCRIPTION
**Pull request by Sridhar-Durgam2022**

### CODE REVIEW

File: src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.apache.flink.table.types.logical.RowType;
+import org.apache.flink.table.types.logical.RowType.RowField;
+import org.junit.Test;
+
+public class ArrowSchemaConverterTest {
+  @Test
+  public void convertToSchemafromRowTypeTest() {
+    ArrowSchemaConverter arrowSchemaConverter = new ArrowSchemaConverter();
+    List<RowField> fields = new ArrayList<RowField>();
+    fields.add(new RowField("id", DataTypes.BIGINT().getLogicalType()));
```
Comment (davidrabinowitz) [inline src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java:35]: Please test all types

File: src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    fields.add(new RowField("name", DataTypes.STRING().getLogicalType()));
+    RowType rowType = new RowType(fields);
+    Schema schema = arrowSchemaConverter.convertToSchema(rowType);
+    assertThat(schema).isNotNull();
+    assertThat(schema.toString()).isEqualTo("Schema<id: Int(64, true), name: Utf8>");
+  }
+
+  @Test
+  public void convertToSchemaToGetFieldTest() {
+    ArrowSchemaConverter arrowSchemaConverter = new ArrowSchemaConverter();
```
Comment (davidrabinowitz) [inline src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java:45]: Please test all types

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]

File: src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java (hunk 1 above)
Comment (Sridhar-Durgam2022) [reply src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java]: Addressed

File: src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java (hunk 2 above)
Comment (Sridhar-Durgam2022) [reply src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java]: addressed

File: src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    fields.add(new RowField("date_dataType", DataTypes.DATE().getLogicalType()));
+    fields.add(new RowField("decimal_dataType", DataTypes.DECIMAL(10, 0).getLogicalType()));
+    fields.add(new RowField("double_dataType", DataTypes.DOUBLE().getLogicalType()));
+    fields.add(new RowField("row_dataType", DataTypes.ROW().getLogicalType()));
+    fields.add(new RowField("timestamp_dataType", DataTypes.TIMESTAMP().getLogicalType()));
+    fields.add(new RowField("time_dataType", DataTypes.TIME().getLogicalType()));
+    RowType rowType = new RowType(fields);
+    Schema schema = ArrowSchemaConverter.convertToSchema(rowType);
+    assertThat(schema).isNotNull();
+    assertThat(schema.toString())
```
Comment (davidrabinowitz) [inline src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java:50]: Please test this by querying the fields rather than the toString()

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]
Why do we need to check that all the created values are not null?

Please test using the Schema and Field APIs, not toString() comparison

File: src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java (hunk 3 above)
Comment (Sridhar-Durgam2022) [reply src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java]: Addressed

**Pull request closed without merging**